safenode/src/client/api.rs

```diff
@@ -91,23 +92,31 @@ impl Client {
             match event {
                 ClientEvent::ConnectedToNetwork => {
                     info!("Client connected to the Network.");
+                    println!("Client successfully connected to the Network.");
```

Comment:
user will only see this `println` , so actually only need to check the `logged_connected` flag here?
then don't need to pass the flag down and do other changes?

Reply:
Ah yeh, clients already did it only once